deploy/scripts/aad-register.ps1

```diff
@@ -224,6 +224,7 @@ Function New-ADApplications() {
                     -TenantId $tenantId `
                 }
             catch {
+                Write-Host "An exception was encountered collecting user credentials when registering $($applicationDisplayName): $($_.Exception.Message)"
```

Comment:
Let's avoid passive voice when possible and simplify, e.g.: "Failed collecting user credentials while registering $($applicationDisplayName): $($_.Exception.Message)"

Reply:
Thanks and great callout!  Will update.